packages/itwin/tree-widget/src/tree-widget-react/components/trees/external-sources-tree/ExternalSourcesTree.tsx

```diff
@@ -34,7 +35,9 @@ export function ExternalSourcesTree(props: ExternalSourcesTreeProps) {
       treeName={ExternalSourcesTreeComponent.id}
       getHierarchyDefinition={getDefinitionsProvider}
       selectionMode={props.selectionMode ?? "none"}
-      treeRenderer={(treeProps) => <TreeRenderer {...treeProps} getDecorations={(node) => <Icon href={getIcon(node)} />} />}
+      treeRenderer={(treeProps) => (
+        <TreeRenderer {...treeProps} actions={props.actions} getDecorations={props.getDecorations ?? ((node) => <ExternalSourcesTreeIcon node={node} />)} />
```

Comment:
I guess you shouldn't pass `actions` to the `Tree` component if they are used by `TreeRenderer`

Reply:
adjusted